index.html

```diff
@@ -1,10 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="css/styles.css" rel="stylesheet" />
-    <title>Document</title>
-  </head>
-  <body></body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="css/styles.css" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300;400;700&display=swap" rel="stylesheet">
+  <title>ÉLECTRO/2019</title>
+</head>
+
+<body>
+
+  <!------------ Background Gradiant ------------>
+  <div class="ej2">
+  </div>
+
+  <!------------ Main Flex Container ------------>
+  <div class="flex-container">
+
+    <!------------ Background EJ ------------>
+    <div class="bgimg"></div>
+
+    <!------------ Red Letters ------------>
+    <h1 class="electro-jazz">
+      <div>
+        É
+      </div>
+      <div>
+        L
+      </div>
+      <div>
+        E
+      </div>
+      <div>
+        C
+      </div>
+      <div>
+        T
+      </div>
+      <div>
+        R
+      </div>
+      <div>
+        O
+      </div>
+      <div>
+        /
+      </div>
+      <div>
+        2
+      </div>
+      <div>
+        0
+      </div>
+      <div>
+        1
+      </div>
+      <div>
+        9
+      </div>
+    </h1>
+
+
+    <!------------ Footer Logo ------------>
+    <div class="footer">
+      <h2>
+        ÉLECTRO <br> JAZZ
```

Comment:
You can skip this `br` element and instead control the width via CSS: `max-width: min-content;` on this `h2` would get you the same effect with more flexibility:
```suggestion
        ÉLECTRO JAZZ
```